src/main/webapp/app/notification/course-notification/course-notification-overview/course-notification-overview.component.scss

```
@@ -79,6 +105,8 @@ $overlay-offset-top: 2.8rem;
     right: 0;
     display: none;
     background-color: var(--module-bg);
+
+    color: var(--bs-body-color);
```

Stylelint (17.13.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)